packages/react-tabster/etc/react-focus-management.api.md

```diff
@@ -4,41 +4,41 @@
 
 ```ts
 
-import { getAbilityHelpersAttribute } from 'ability-helpers';
+import { getTabsterAttribute } from 'tabster';
 import * as React from 'react';
-import { Types } from 'ability-helpers';
+import { Types } from 'tabster';
+
+export { getTabsterAttribute }
+
+// @public (undocumented)
+export const renderTabsterProvider: (state: TabsterProviderState) => JSX.Element;
 
 // @public (undocumented)
-export interface FocusManagementProvideProps extends React.HTMLAttributes<HTMLElement> {
+export interface TabsterProvideProps extends React.HTMLAttributes<HTMLElement> {
     customRoot?: boolean;
-    // (undocumented)
-    dir?: 'ltr' | 'rtl';
-    document?: Document;
 }
 
 // @public
-export const FocusManagementProvider: React.FunctionComponent<FocusManagementProvideProps>;
+export const TabsterProvider: React.FunctionComponent<TabsterProvideProps>;
 
 // @public (undocumented)
-export interface FocusManagementProviderState extends FocusManagementProvideProps {
-    // Warning: (ae-forgotten-export) The symbol "FocusManagementContextValue" needs to be exported by the entry point index.d.ts
+export interface TabsterProviderState extends TabsterProvideProps {
+    // Warning: (ae-forgotten-export) The symbol "TabsterContextValue" needs to be exported by the entry point index.d.ts
     //
     // (undocumented)
-    contextValue: FocusManagementContextValue;
+    contextValue: TabsterContextValue;
     // (undocumented)
-    dir: FocusManagementProvideProps['dir'];
+    dir: 'ltr' | 'rtl';
+    // (undocumented)
+    document?: Document;
 }
 
-export { getAbilityHelpersAttribute }
-
-// @public (undocumented)
-export const renderFocusManagementProvider: (state: FocusManagementProviderState) => JSX.Element;
-
 // @public
-export const useArrowNavigationGroup: (options?: UseArrowNavigationGroupOptions) => Types.AbilityHelpersDOMAttribute;
+export const useArrowNavigationGroup: (options?: UseArrowNavigationGroupOptions | undefined) => Types.TabsterDOMAttribute;
```

Comment:
[non-blocking] 
is `| undefined` really needed if options is already optional?

doesn't `?` allow it to be omitted or undefined already?

Reply:
It does, I'll remove it!

Reply:
Seems like that only exists in the API declaration, which is automatically generated.
![image](https://user-images.githubusercontent.com/5483269/113282921-e14db900-92e7-11eb-9c3d-9477ce3f5d40.png)